docker/test/stateless/setup_export_logs.sh

```diff
@@ -0,0 +1 @@
+../../../utils/export-logs-in-ci/setup.sh
```

Comment:
It shouldn't work. Does it work?

Docker usually rejects copying anything out of the context directory, which is `
docker/test/stateless` in this case

Reply:
Ah, ok. It works now but will break after the #52865 will be merged. The `setup.sh` should be in the docker directory and linked to utils.

Reply:
Ok. I don't need the `utils/export-logs-in-ci/setup.sh` script, but I want it in almost every Docker image: stateless, stateful, stress, fuzzer... How to do it?

Reply:
If looks like you need to put it into base https://github.com/ClickHouse/ClickHouse/blob/master/docker/images.json#L120

Or, if in fasttests too, than to util https://github.com/ClickHouse/ClickHouse/blob/master/docker/images.json#L120

Reply:
I will put it to `base`